docs/security-considerations.rst

```diff
@@ -35,14 +35,19 @@ As always, with open source documentation, please help us extend this section
 Pitfalls
 ********
 
-Private Information and Randomness
-==================================
+Private Information
+===================
 
 Everything you use in a smart contract is publicly visible, even
 local variables and state variables marked ``private``.
 
-Using random numbers in smart contracts is quite tricky if you do not want
-miners to be able to cheat.
+Randomness
+==========
+
+Using random numbers in smart contracts is tricky if you don't want
+miners to be able to cheat. Getting randomness right is often the crucial part in a crypto project and most failures result from bad random number generators.
+
+If you don't want your contract to be safe, try something like the `coin flipper <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/35_coin_flipper.sol>`_ or use a contract that supplies randomness, like the `RANDAO <https://github.com/randao/randao>`_.
```

Comment:
Not sure if we should mention this.

Reply:
@chriseth You are the expert, if you don't think we should mention it, I'll cut it. But do you mean the concept or the examples, or both?

Reply:
Just this paragraph.